doc/installation/setup.md

```diff
@@ -4,29 +4,29 @@
 
 #### Install `nix` package manager
 
-If you have a pass Culture's computer, which has a proxy that adds a custom certificate, the install may fail
+If you have a pass Culture's computer, which has a proxy that adds a custom certificate, the install may fail.
 
-In that case, you will need to install Nix as follows
+In that case, you will need to install Nix as follows :
 
 ```sh
 curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install  --ssl-cert-file '/Library/Application Support'/*/*/data/*cacert.pem
 ```
 
-If you want more information or if you have a problem you can consult [`nix` package manager installation](https://github.com/DeterminateSystems/nix-installer#the-determinate-nix-installer)
+_If you want more information or if you have a problem you can consult [`nix` package manager installation](https://github.com/DeterminateSystems/nix-installer#the-determinate-nix-installer)._
 
 #### Install DirEnv
 
+1.
+
 ```sh
 brew install direnv
 ```
 
-If you want more information or if you have a problem you can consult [DirEnv installation](https://direnv.net/)
-
-You will need to add [hook into your user configuration (example: `~/.zshrc`)](https://direnv.net/docs/hook.html)
+_If you want more information or if you have a problem you can consult [DirEnv installation](https://direnv.net/)._
 
```

Comment:
pour mettre des trucs dans une liste, il faut l'indenter

~~~suggestion
1. Install the executable

   ```sh
   brew install direnv
   ```

   _If you want more information or if you have a problem you can consult [DirEnv installation](https://direnv.net/)._
~~~